CHANGELOG.md

```diff
@@ -82,6 +82,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -	Fixed: Added Invisible Objects and Dark Visor Requirements for Screw Attack without Space Jump in Phazon Grounds (Hard and above).
 
+-   Added: Path of Roots missile using only Morph, Bombs and Space Jump.
```

Comment:
I would recommend adding the difficulty this trick is used in, for clarity. See above lines for examples on this.